notebooks/community/model_evaluation/automl_tabular_regression_model_evaluation.ipynb

```diff
@@ -0,0 +1,1567 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.        dataflow_machine_type: str = 'n1-standard-4',
</pre><p>Since we are not modifying any dataflow parameter, we can remove them all.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>